tRecorderApi/api/models/take.py

```diff
@@ -33,3 +48,141 @@ def get_takes(chunk_id):
             }
             ls.append(tk)
         return ls
+
+    @staticmethod
+    def saveTakesToDB(meta, relpath, take_data, manifest, published=False):
+        try:
+            # Create Language in database if it's not there
+            language_obj, l_created = Language.objects.get_or_create(
+                slug=manifest["language"]["slug"],
+                defaults={
+                    'slug': manifest["language"]["slug"],
+                    'name': manifest["language"]["name"]},

```

Comment:
closing curly brace should be on the next line, like in line 68 just a bit further down